platform_global_teacher_campus/api/v1/serializers.py

```diff
@@ -10,6 +10,7 @@
     ValidationRejectionReason,
     ValidationRules,
 )
+from rest_framework.permissions import IsAuthenticated
```

Comment:
Why did you import this here?

Reply:
Is to enforce that the user making the request must be authenticated